### ISSUE TITLE
add link preview metadata showing map

### ISSUE THREAD

**Issue by hb0nd** [organicmaps/url-processor#5]
Currently we have no metadata:
![Screenshot_20230315-103809_WhatsApp_1](https://user-images.githubusercontent.com/26939824/225285491-17e5502b-20b6-42b9-9178-65589d60840c.png)

**Comment by hb0nd** [comment]
can test with https://github.com/rafaelmardojai/share-preview

**Comment by muellerphilipp** [comment]
Is that feature still of interest? I'd like to take a look at it if it's still wanted.

**Comment by biodranik** [comment]
Yes, a proper map image highlighting the shared place should be generated and stored somewhere to be used in the preview. Do you have an idea how to do that?

**Comment by muellerphilipp** [comment]
OK, thanks! I will familiarise myself with the repo and get back to you when I have an idea of how to deal with it.
